### PR TITLE
Inital update of README - more work to be done

### PR DESCRIPTION
**Pull request by shymega**
Signed-off-by: Dom Rodriguez <shymega@shymega.org.uk>

## Contributor Checklist:

* [ ] I have created a file in the `doc/newsfragments` directory *IF* it is a
      user-visible change (and make sure to read the `README.md` in that directory)

### CODE REVIEW

**Comment by shymega** [comment]
I'm in doubt if I need to add a newsfragment.

**Comment by shymega** [comment]
Can squash if needed, Not happy with the commit message either.

**Review by p12tic — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,99 +1,116 @@
...
 
 ### Usage
 
-Install and run barrier on each machine that will be sharing.
-On the machine with the keyboard and mouse, make it the server.
+1. Install and run Input Leap on each machine that will be sharing.
+2. On the machine with the keyboard and mouse, make it the server.
+3. Click the "Configure server" button and drag a new screen onto the grid for each client machine.
+4. Ensure the "screen name" matches exactly (case-sensitive) for each configured screen -- the clients' Input Leap windows will tell you their screen names (just
+above the server IP).
```
Comment (p12tic) [inline README.md:60]: Bad line break.

File: README.md [hunk 2]
```diff
@@ -1,99 +1,116 @@
...
-Note that if the keyboard's Scroll Lock is active then this will prevent the mouse from switching screens.
+Note that if the keyboard's Scroll Lock is active then this will prevent the
+mouse from switching screens.
 
 ### Contact & support
 
-Please be aware that the *only* way to draw our attention to a bug is to create a new issue in [the issue tracker](https://github.com/debauchee/barrier/issues). Write a clear, concise, detailed report and you will get a clear, concise, detailed response. Priority is always given to issues that affect a wider range of users.
+Please be aware that the *only* way to draw our attention to a bug is to create
+a new issue in [the issue
+tracker](https://github.com/input-leap/input-leap/issues).
```
Comment (p12tic) [inline README.md:75]: bad line break.

**Pull request merged**